mava/wrappers/jaxmarl.py

```diff
@@ -163,36 +163,36 @@ def __init__(self, env: MultiAgentEnv, timelimit: int = 500):
         else:
             self.get_avail_actions = lambda _: jnp.ones(self._action_shape)
 
-    def reset(self, key: PRNGKey) -> Tuple[JaxMarlState, TimeStep[Observation]]:
+    def reset(self, key: PRNGKey) -> Tuple[JaxMarlState, TimeStep[ObservationGlobalState]]:
         key, reset_key = jax.random.split(key)
         obs, state = self._env.reset(reset_key)
 
-        extras = {"global_state": jnp.array(obs["world_state"])}
-        obs = Observation(
+        obs = ObservationGlobalState(
             agents_view=batchify(obs, self.agents),
             action_mask=self.get_avail_actions(state),
+            global_state=self.get_global_state(obs),
             step_count=jnp.zeros(self._env.num_agents, dtype=int),
         )
-        return JaxMarlState(state, key, 0), restart(obs, extras=extras, shape=(self.num_agents,))
+        return JaxMarlState(state, key, 0), restart(obs, extras={}, shape=(self.num_agents,))
```

Comment:
extras default is `{}`
```suggestion
        return JaxMarlState(state, key, 0), restart(obs, shape=(self.num_agents,))
```

Reply:
The default is None which causes an error for the type change: